data/posts/react-router-brand-update.md

```diff
@@ -0,0 +1,73 @@
+
+<img alt="The new React Router logo showing what has changed" src="/blog-images/posts/react-router-brand-update/logo-after.png" class="m-auto w-4/5 border rounded-md" />
+
+## Updating the wordmark
+
+When it came to this piece of the puzzle, the team had a greater appetite for something new. Ever since posting about [merging Remix and React Router](https://remix.run/blog/merging-remix-and-react-router) in May, we've been toying with the idea of merging the brands together visually. In particular, we wondered whether this was an opportunity for the Remix "R" to be integrated, considering the way we've used Founders Grotesk Bold as the logo font for both identities in the past. 
+
+Ultimately, we decided against visually integrating it. The Remix brand will stay separate, free to evolve as we awaken from our nap in 2025 to release new Remix packages. By making this decision, the React Router wordmark wouldn't depend on another logo (and wouldn't limit this exploration either). It allowed us to explore something more specific to React Router. 
+
+<img alt="Three iterations of the React Router wordmark" src="/blog-images/posts/react-router-brand-update/wordmark-iterations.png" class="m-auto w-4/5 border rounded-md" />
+
+Another problem was simply that the words "React Router" were quite long when included in a brand lockup. Instead of creating various vertically and horizontally stacked configurations of the Logo + Wordmark, we wondered if we could provide a single, evergreen solution.
+
+Inspired by the ancient "compact disc" logo, we embraced a method of nesting the word "React" and emphasizing "Router" as the more significant idea. Visually, this was an elegant way of reconciling the long width issue while establishing a more accurate hierarchy for the identity.
+
+I kept some usage of Founders Grotesk and created a custom ligature that more comfortably allowed for nesting. Connecting the "u" with the "t" is also a visual reference to the way nodes connect in the logo.
+
+<img alt="A closeup of the React Router wordmark showing the custom ligature" src="/blog-images/posts/react-router-brand-update/wordmark.png" class="m-auto w-4/5 border rounded-md" />
+
+<img alt="Stylistic variants for the React Router wordmark in black and white" src="/blog-images/posts/react-router-brand-update/wordmark-variants.png" class="m-auto w-4/5 border rounded-md" />
+
+When the word "Router" is styled as an outline, you can more clearly see the retro inspiration behind it.
+
+## Creating a mark for this release
+
+Initially, I didn't think we needed a unique visual identifier for "v7." After all, React Router has been around for years and previous versions never received a visual identity of their own. But a release such as this is significant to those 7+ million React Router projects (and the 5 million line Shopify App) and we take our work very seriously. 
```

Comment:
> 7+ million

9 million on GitHub now:

<img width="350" alt="image" src="https://github.com/user-attachments/assets/0c761b80-346c-44f4-94e1-18988633b760">


I think you could just say the millions. It's so hard to quantify this stuff, especially because this doesn't capture private repos, and it does capture numerous demo apps that don't _really_ count

Reply:
Ah, great to know! (9.1million) Up 29% from ~7million is non-trivial, so "millions" doesn't quite have the same impact. Then again, I'm not trying to inflate the significance of the point using the difficult-to-quantify demo apps either, so perhaps "millions" is the better edit.